### DISCUSSION TITLE
Should the Thread radio be disabled by default?

### DISCUSSION THREAD

**Discussion by zhanglongxia** [openthread/openthread#10553] [General]
When the Thread stack starts, it sets the radio to be enabled by default ([mac.cpp](https://github.com/openthread/openthread/blob/main/src/core/mac/mac.cpp#L122)). The radio will always be enabled unless users manually call the cli command `radio disable`. For the POSIX platform, the [radio_spinel.cpp](https://github.com/openthread/openthread/blob/main/src/lib/spinel/radio_spinel.cpp#L1888) periodically send Spinel command `SPINEL_PROP_RCP_TIMESTAMP` to RCP to synchronize time. If Thread stack is not enabled, it still periodically wake up Thread host. I want to stop the time sync when the Thread stack is not enabled to save power. Could we set the Thread radio to be disabled by default, enable Thread radio when the [Thread network interface is up](https://github.com/openthread/openthread/blob/main/src/core/thread/thread_netif.cpp#L61) and disable Thread radio when the [Thread network interface is down](https://github.com/openthread/openthread/blob/main/src/core/thread/thread_netif.cpp#L87)? So that the `radio_spinel.cpp` can start or stop the RCP time sync based on the radio's state.

**Comment by abtink** [comment]
> Could we set the Thread radio to be disabled by default, enable Thread radio when the [Thread network interface is up](https://github.com/openthread/openthread/blob/main/src/core/thread/thread_netif.cpp#L61) and disable Thread radio when the [Thread network interface is down](https://github.com/openthread/openthread/blob/main/src/core/thread/thread_netif.cpp#L87)?

While I like this idea and I think it seems reasonable, I suggest that we'd be careful about this. Modifying this behavior could have unintended consequences for existing code that integrate the OpenThread stack. Some app or platform-layer code might implicitly depend on the fact that radio being enabled upon OpenThread stack initialization (from the `Mac` constructor). 

I recommend maybe exploring a `RadioSpinel`-specific solution first, before making any changes to the core stack behavior.
